hudi-common/src/main/java/org/apache/hudi/avro/HoodieAvroUtils.java

```diff
@@ -1379,6 +1379,88 @@ public static boolean hasListOrMapField(Schema schema) {
     }
   }
 
+  public static void validateRecordsHaveSameData(Object expected, Object actual) {
```

Comment:
This is used by test code, can we move it to a test class?

Reply:
done